src/features/user/TreeMapper/components/PlantLocationPage.tsx

```diff
@@ -174,11 +174,11 @@ export function LocationDetails({
       <div className={styles.details}>
         <div className={styles.singleDetail}>
           <p className={styles.title}>{t('captureMode')}</p>
-          <div className={styles.value}>{location.captureMode}</div>
+          <div className={styles.value}>{t(location.captureMode)}</div>
```

Comment:
There ain't any resources named `location.captureMode` or `location.captureStatus` yet, are there?

Reply:
sorry, this is ok